Source/FindInBlueprintExternal/Private/FiBCommandlet.cpp

```diff
@@ -0,0 +1,43 @@
+#include "FiBCommandlet.h"
+#include "AssetRegistryModule.h"
+#include "Developer/AssetTools/Public/AssetToolsModule.h"
+#include "FindInBlueprintManager.h"
+//#include "SharedPointer.h"
+
+int32 UFiBCommandlet::Main(const FString& Params)
+{
+	const TCHAR* Cmd = *Params;
+	GIsRunning = true;
+
+	FAssetRegistryModule* AssetRegistryModule = &FModuleManager::LoadModuleChecked<FAssetRegistryModule>(TEXT("AssetRegistry"));
```

Comment:
We get a reference back from this ftn so just assign to a reference
```suggestion
	FAssetRegistryModule& AssetRegistryModule = FModuleManager::LoadModuleChecked<FAssetRegistryModule>(TEXT("AssetRegistry"));
```

Reply:
Assigning to a pointer we lose the inherent safety of a reference, and there's no need for that here.